LICENSE

```diff
@@ -192,7 +192,7 @@
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
 
-       http://www.apache.org/licenses/LICENSE-2.0
+       https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This was from https://www.apache.org/licenses/LICENSE-2.0, we should not change here.

Reply:
Done